napari/_qt/qt_resources/styles/02_custom.qss

```diff
@@ -267,12 +267,15 @@ QLabel#logo_silhouette {
 
 /* ------------------------------------------------------ */
 
+QFrame#empty_controls_widget {
+    min-height: 295px;
```

Comment:
```suggestion
    min-height: 225px;
```
What do you think of making the empty height based of the Shapes layer which has the smallest Layer Controls widget?
Then every other one resizes the widget larger.
Note: they don't shrink back, once resized larger that becomes the new min, but that's fine? 🤷‍♂️

Reply:
Actually Image is the smallest, but then it's like 180.

Reply:
> What do you think of making the empty height based of the Shapes layer which has the smallest Layer Controls widget?

I'm not totally sure from where the min values come from (I guess at some point those looked good in a standard setup on MacOS running a specific example code or for the most used type of layer?)  but decreasing it a little bit shouldn't be a problem with the changes here so I will apply the suggestion :+1:

> Note: they don't shrink back, once resized larger that becomes the new min, but that's fine? 🤷‍♂️
Maybe calling `adjustSize` when setting the current widget could help, will test :+1:

Reply:
The old min was exactly labels layer controls in 2D on my Mac. Labels has the biggest widget.

Reply:
Did a quick check with `adjustSize` but seems like that doesn't work well (probably needs also a call from the dockwidget) 🤔 If there is no problem with resize large becoming new min then the changes here should be okay

Reply:
Yup! An improvement for sure!
You're a wizard! 🧙‍♂️